static/monsterSort.js

```diff
@@ -0,0 +1,16 @@
+$(document).ready(function() {
+	$('select').change(function() {
+		selection = $(this).val();
+		if (selection == 'all') {
+			$('.monster-box').show();
+		} else {
+			var selector = '.' + selection;
```

Comment:
should i change it to ".monster-type-[good|evil|conflicted]" or something like that?